scala/scala_2.13/src/test/scala/com/linecorp/armeria/common/RequestContextSuite.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import com.linecorp.armeria.client.ClientRequestContext
+import com.linecorp.armeria.scala.implicits._
+import munit.FunSuite
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+class RequestContextSuite extends FunSuite {
+  test("RequestContext.eventLoopExecutionContext") {
+    val ctx = ClientRequestContext.of(HttpRequest.of(HttpMethod.GET, "/"))
+    implicit val ec: ExecutionContext = ctx.eventLoopExecutionContext
+    val invoked = new AtomicBoolean()
+
+    Await.ready(
+      Future {
+        assert(ClientRequestContext.current == ctx)
+        invoked.set(true)
+      },
+      10.seconds)
```

Comment:
We could remove `Await.ready()` because the returned `Future` will automatically be awaited by munit. https://scalameta.org/munit/docs/tests.html#declare-async-test

Reply:
But I have to check if `invoked.set(true)` has been invoked after `ready()`.

Reply:
I thought we don't need to check `invoked` because if the assert fails, this test fails too.